DESCRIPTION

```diff
@@ -7,11 +7,10 @@ Authors@R: c(
                     email = "boris.steipe@utoronto.ca", 
                     role = c("aut", "cre"), 
                     comment = c(ORCID = "0000-0002-1134-6758")),
-             person("Rachel", "Silverstein", 
```

Comment:
This is actually a merge conflict - and I don't know why it was not flagged as such. Did you delete Rachel's entry? Anyway: update DESCRIPTION to the latest version before we can check this in.

Reply:
resolved conflict